src/transformStateWithClones.js

```diff
@@ -8,6 +8,40 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  let stateClone = { ...state };
+  const stateTransform = [];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        stateClone = { ...stateClone };
```

Comment:
I don't think this reassign is necessary

Reply:
I agree with @Alireza-Mohammadhossein